src/fsharp/XmlDoc.fs

```diff
@@ -146,10 +147,10 @@ and XmlDocStatics() =
 /// Used to collect XML documentation during lexing and parsing.
 type XmlDocCollector() =
     let mutable savedLines = ResizeArray<string * range>()
-    let mutable savedGrabPoints = ResizeArray<pos>()
+    let mutable savedGrabPoints = Dictionary<_, _>()
```

Comment:
Please either specify `Dictionary` type arguments or remove the type argument list.